TKGame/Level Editor Content/Stages/room2.json

```diff
@@ -87,6 +100,13 @@
       "width": 10,
       "height": 195,
       "action": "goPrevious"
+    },
+    {
+      "X": 1590,
+      "Y": 660,
+      "width": 10,
+      "height": 195,
+      "action": "goNext"
```

Comment:
What does this mean?

Reply:
This is a trigger or a door basically. the action is a simple string that is used for determining if you stage to be loaded should be currentstage + 1 or - 1.